main.tf

```diff
@@ -98,6 +99,31 @@ module "nat_instances" {
   vpc_id                  = module.vpc.vpc_id
 }
 
+# ---------------------------------------------------------------------------------------------------------------------
+# Use of alterNAT instances instead of NAT instances to reduce costs on the basis of hourly usage.
+# ---------------------------------------------------------------------------------------------------------------------
+module "alternat_instances" {
+  count = local.create_alter_nat ? 1 : 0
+
+  source = "git::https://github.com/1debit/alternat.git//modules/terraform-aws-alternat?ref=v0.1.0"
+
+  alternat_image_uri = "531874807515.dkr.ecr.eu-central-1.amazonaws.com/alternat"
+  alternat_image_tag = "latest"
+
+  ingress_security_group_ids = var.ingress_security_group_ids
+
+  subnet_suffix     = var.nat_subnet_suffix
+  nat_instance_type = "t4g.nano"
+
+  private_route_table_ids = module.vpc.private_route_table_ids
+
+  tags = var.tags
+
+  vpc_id                 = module.vpc.vpc_id
```

Comment:
Please also add the case when VPC creation is deactivated and a VPC is already given. Currently, these lines indicate that only a Terra3-created VPC is supported